sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -456,8 +456,8 @@ def to_sql(self) -> str:
     def to_bigquery(
         self,
         job_config: Optional[bigquery.QueryJobConfig] = None,
-        timeout: int = 1800,
-        retry_cadence: int = 10,
+        timeout: Optional[int] = None,
```

Comment:
can you keep the default value? otherwise, this might become a breaking change

Reply:
done